services/WorkflowsService.js

```diff
@@ -50,18 +66,23 @@ exports.workflowsDELETE = async function (args, res, next) {
       workflowName,
       "json"
     );
-    fileExists? await DataHandlerFileSystem.removeFile(
+    fileExists
+      ? await DataHandlerFileSystem.removeFile(
           workflowsPath,
           workflowName,
           "json"
         )
       : res.end("Not found");
 
-      //TODO: add logger
+    logger.info(
```

Comment:
logger.info with request type and args before doing an action

Reply:
fixed in 0433386